Source/ZoomNet/Models/ReportParticipant.cs

```diff
@@ -0,0 +1,103 @@
+using Newtonsoft.Json;
+using System;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Metrics of a participant.
+	/// </summary>
+	public class ReportParticipant
+	{
```

Comment:
Also, double check if the properties of a webinar report participant are the same as a meeting report participant. If they are different, we'll need ReportMeetingParticipant and ReportWebinarParticipant classes.